toolkit/tools/pkg/licensecheck/testdata/README.md

```diff
@@ -0,0 +1,33 @@
+# Test data for the license checker
```

Comment:
Markdown files should also be human readable. So, it is goodness to line wrap (at 88).

Reply:
Ah, didn't even consider it. My editor automatically wraps markdown.